evm_scilla_js_tests/test/scilla/Timestamp.ts

```diff
@@ -12,7 +12,7 @@ xdescribe("Scilla timestamp #parallel", () => {
     const blockCount = await ethers.provider.getBlockNumber();
     const blockTimestamp = (await ethers.provider.getBlock(blockCount)).timestamp;
     const tx = await contract.EventTimestamp(blockCount);
-    const timestamp = Number(tx.getReceipt().event_logs[0].params[0].value.arguments[0]);
+    const timestamp = Number(JSON.parse(tx.receipt.event_logs[0].params[0].value).arguments[0]);
```

Comment:
Can you compare the responses for ZQ1 and ZQ2 before we merge this change? The fact that we needed to add a `JSON.parse` suggests we might have changed the format? (double encoding?)

Reply:
Good point! :) I was also suspicious of this new requirement.

Reply:
I created #1380 for it. I know the cause and I have a solution but it needs more tests so let's merge this ticket first then I'll raise another PR.